README.rst

```diff
@@ -1422,6 +1463,74 @@ Example usage:
         '*png',
     ]
 
+``OPENWISP_HTML_EMAIL``
+^^^^^^^^^^^^^^^^^^^^^^^
+
++---------+----------+
+| type    | ``bool`` |
++---------+----------+
+| default | ``True`` |
++---------+----------+
+
+If ``True``, attaches markdown rendered HTML of message in email.
```

Comment:
it's mentioning markdown but I don't think we are using markdown in this module (it's done in openwisp-notifications).

Let's just say that if True an HTML themed version of the email can be sent using the send_email function (let's link to the section which explains send_email).